src/test/java/uk/gov/hmcts/reform/civil/utils/HearingUtilsTest.java

```diff
@@ -63,20 +67,55 @@ void shouldReturnFee7_whenGivenAnyClaimFee() {
     }
 
     @Test
-    void shouldReturnFormattedFee_whenGivenAnyClaimFee() {
+    void shouldReturnFormattedFee_whenGivenMidClaimFee() {
         assertThat(HearingUtils.formatHearingFee(
             Fee.builder().calculatedAmountInPence(new BigDecimal(34600)).build())).isEqualTo("£346");
     }
 
     @Test
-    void shouldReturnFormattedFee2_whenGivenAnyClaimFee() {
+    void shouldReturnFormattedFee_whenGivenBigClaimFee() {
         assertThat(HearingUtils.formatHearingFee(
             Fee.builder().calculatedAmountInPence(new BigDecimal(132000)).build())).isEqualTo("£1,320");
     }
 
     @Test
-    void shouldReturnFormattedFee3_whenGivenAnyClaimFee() {
+    void shouldReturnFormattedFee_whenGivenSmallClaimFee() {
         assertThat(HearingUtils.formatHearingFee(
             Fee.builder().calculatedAmountInPence(new BigDecimal(5000)).build())).isEqualTo("£50");
     }
+
+    @Test
+    void shouldReturnNull_when0ClaimFee() {
+        assertThat(HearingUtils.formatHearingFee(
+            Fee.builder().calculatedAmountInPence(new BigDecimal(0)).build())).isNull();
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {34600, 132000, 5000})
+    void shouldReturnFormattedFee_whenGivenAnyClaimFee(int amount) {
+        BigDecimal feeAmount = new BigDecimal(amount);
+        assertThat(HearingUtils.formatHearingFee(
+            Fee.builder().calculatedAmountInPence(feeAmount).build())).isNotEmpty();
+    }
+
+    @ParameterizedTest
+    @EnumSource(HearingDuration.class)
+    void shouldReturnHearingDuration_whenGivenAnyHearingDuration(HearingDuration hearingDuration) {
+        assertThat(HearingUtils.formatHearingDuration(hearingDuration)).isNotEmpty();
+    }
+
+    @Test
+    void shouldReturnNull_whenNotAllowedTime() {
+        assertThat(HearingUtils.getHearingTimeFormatted("50000")).isNull();
+    }
+
+    @Test
+    void shouldReturnNull_whenGivenEmptyTime() {
+        assertThat(HearingUtils.getHearingTimeFormatted("")).isNull();
+    }
+
+    @Test
+    void shouldReturnTimedFormatted_whenGivenAnyTime() {
+        assertThat(HearingUtils.getHearingTimeFormatted("0500")).isEqualTo("05:00");
+    }
```

Comment:
One of the points of me asking for tests on this method was to make sure we check boundary conditions. In this particular test there's two (plus one) boundary conditions we need to make sure will be interpreted correctly:
- The handling of 12:00 as it could be misinterpreted as midnight (00:00)
- the check for hours above 12:00 as we want to be sure we are in 24h format
- the handling of non-whole hours to make sure minutes are considered too (just to be thorough)

Consider this test:
```
    @ParameterizedTest
    @CsvSource(value = {"0000;00:00", "0500;05:00", "1200;12:00", "2300;23:00", "1230;12:30", "2318;23:18"}, delimiter=';')
    void shouldReturnTimedFormatted_whenGivenAnyTime(String input, String expectedOutput) {
        assertThat(HearingUtils.getHearingTimeFormatted(input)).isEqualTo(expectedOutput);
    }
```